lib/sentry/client.ex

```diff
@@ -157,6 +163,22 @@ defmodule Sentry.Client do
     {endpoint, public_key, secret_key}
   end
 
+  def maybe_call_after_send_event({:ok, _} = result, event) do
+    case Application.get_env(:sentry, :after_send_event) do
```

Comment:
Can probably break this case into a function and use it for both scenarios instead of copying it.